sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/DiagnosticProperty.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Messaging.ServiceBus.Diagnostics
+{
+    /// <summary>
+    ///   The set of well-known properties associated with Service Bus diagnostics.
+    /// </summary>
+    ///
+    internal static class DiagnosticProperty
+    {
+        /// <summary>The common root for activity names in the Service Bus context.</summary>
+        public const string BaseActivityName = "Azure.Messaging.ServiceBus";
```

Comment:
This was just refactored in Event Hubs.... you may want to sync it up.   Names were tightened per UX study feedback.  I think the entire concept of `BaseActivityName` is no longer a thing, IIRC.

Reply:
Covered in https://github.com/Azure/azure-sdk-for-net/issues/9881